docs/README.md

```diff
@@ -1,105 +1,411 @@
-# Geolocation GraphQL Interface
+# Geolocation GraphQL interface
 
-This app exports a GraphQL schema for geolocation results for Place Components.
+>⚠️ **USE AT YOUR OWN RISK**
+> 
+> *This is an ongoing, unsupported, unfinished and undocumented project. We do not guarantee any results after installation.*
 
-## Usage
+The Geolocation GraphQL interface provides autosuggestions for address inputs and autocompletion for address forms.
 
-### Installation
+>ℹ️ The Geolocation GraphQL interface allows you to choose which geolocation provider (e.g., [Google Maps](https://www.google.com.br/maps/), [Algolia Places](https://community.algolia.com/places/), [VTEX Places](https://github.com/vtex-apps/places-graphql), etc.) suits better for your store.
```

Comment:
VTEX Places actually refers to https://github.com/vtex/places, which is an API that get address data from VTEX database. As it is a private project, I'm not sure how to link it